apps/comps/components/create-agent/agent-created.tsx

```diff
@@ -11,10 +11,10 @@ import { CopyButton } from "@/components/copy-button";
 import { socialLinks } from "@/data/social";
 import { useSession } from "@/hooks/useSession";
 import { useUnlockKeys } from "@/hooks/useUnlockKeys";
-import { Agent } from "@/types";
+import type { RouterOutputs } from "@/rpc/router";
```

Comment:
It's never been clear to me if it is necessary to use `import type` when the think you are importing is already a type. Just thinking out loud here.